rse-py-package.Rmd

```diff
@@ -0,0 +1,1460 @@
+# Python Packaging {#rse-py-package}
+
+## Questions {#rse-py-package-questions}
+
+-   How can I manage the packages my project relies on?
+-   How can I package up my work for others to use?
+-   How should I announce my work?
+
+## Objectives {#rse-py-package-objectives}
+
+```{r, child="objectives/rse-py-package.md"}
+```
+
+## Introduction {#rse-py-package-intro}
+
+> Another response of the wizards,
+> when faced with a new and unique situation,
+> was to look through their libraries to see if it had ever happened before.
+> This was...a good survival trait.
+> It meant that in times of danger you spent the day sitting very quietly in a building with very thick walls.
+>
+> -- Terry Pratchett
+
+The more software you write,
+the more you realize that a programming language is a way to build and combine software libraries.
+Every widely-used language now has an online repository
+from which people can download and install those libraries.
+This lesson shows you how to use Python's tools to create and share libraries of your own.
+
+This material is based in part on [Python 102][python-102] by [Ashwin Srinath][srinath-ashwin].
+
+## How can I turn a set of Python source files into a module? {#rse-py-package-modules}
+
+Any Python source file can be imported by any other.
+When a file is imported,
+the statements in it are executed as it loads.
+Variables defined in the file are then available as `module.name`.
```

Comment:
Where does the `module` part of the name come from?

Reply:
Clarified this